src/main/java/org/gridsuite/securityanalysis/server/entities/ContingencyElementEmbeddable.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.securityanalysis.server.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+import com.powsybl.contingency.ContingencyElementType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/**
+ * @author Laurent GARNIER <laurent.garnier at rte-france.com>
+ */
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Embeddable
+public class ContingencyElementEmbeddable {
```

Comment:
perhaps remove `Embeddable` in the name ?
technical term ?

Reply:
I followed what was previously done in such a case, like ComponentResultEmbeddable in study-server or ReportValueEmbeddable in report-server.
It has some consistency with suffixing with Entity, Repository, Infos, Attributes which also are redundant with package name.
Java is less practical in this than C++ with namespaces. In java, the choice in just twofolds in this respect : either just (short) name of the class or the full (packaged) name.
An intermediate possibility involve static inner classes, but here I do not see usable.

Reply:
ok